docs/book/application-integration/usage-in-a-laminas-mvc-application.md

```diff
@@ -0,0 +1,181 @@
+# Usage in a laminas-mvc Application
+
+The following example shows _one_ potential use case of laminas-form within
+a laminas-mvc based application. The example uses a module, a controller and the
+laminas-form element manager.
+
+The example is based on the [tutorial application](https://docs.laminas.dev/tutorials/getting-started/overview/)
+which builds an album inventory system.
+
+Before starting, make sure laminas-form is installed and configured.
```

Comment:
I'd like to see a link here - to "installation and configuration".

Reply:
At the moment we have no page for this topic. If you remember, we wanted to use the theme for this: https://github.com/laminas/documentation-theme/blob/master/theme/pages/installation.html